backend/.env-example

```diff
@@ -10,7 +10,14 @@ SECRET_MANAGER_SALT=$2b$10$iDQWc/ZxTnAOMdx8022VWe
 JWT_SECRET=secret
 API_KEY_SALT_V1=$2b$10$5CuueB/CRZq8/JgmkzAK1e
 UNSUBSCRIBE_HMAC_ALGO_V1=sha256
-UNSUBSCRIBE_HMAC_KEY_V1=secret
+
```

Comment:
```suggestion
UNSUBSCRIBE_HMAC_KEY_V1=secret
```

Reply:
Sorry what is this change?

Reply:
Hmmm do we still need `UNSUBSCRIBE_HMAC_KEY_V1`?

Reply:
oh ok thanks for the catch